src/platform/unix/BuildLinuxImage.sh.in

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
+# Our sysctl call may be run as normal user,
+# but /sbin is not always in normal users' PATH.
+if [[ -f /sbin/sysct && -x /sbin/sysctl ]]; then sysctl_bin=/sbin/sysctl
```

Comment:
```suggestion
if [[ -x /sbin/sysctl ]]; then sysctl_bin=/sbin/sysctl
```

Looked like `sysct` was a typo. `-x` covers "file exists and execute (or search) permission is granted", so just `-x` should be sufficient.

Reply:
> Looked like `sysct` was a typo. `-x` covers "file exists and execute (or search) permission is granted", so just `-x` should be sufficient.

-x will return true on a directory as well. -f ensures it's a file :)

Reply:
(I don't know how to fix the typo and update the PR)

Reply:
> -x will return true on a directory as well. -f ensures it's a file :)

Good point. I'd never thought of that (or seen a double test) in 40 years of programming shell scripts. TIL.

To fix it, you could just edit the file and add another commit.